client/lib/ffi.rs

```diff
@@ -24,7 +24,7 @@ fn set_last_error(error: Error) {
 /// FFI representation of [super::Error](super::Error)
 ///
 /// The full error can be extracted with get_last_error.
-/// See [super::Error](super::Error) for more details on what these mean.
+/// See [Error](super::Error) for more details on what these mean.
```

Comment:
This should be:
```rust
/// FFI representation of [Error](super::Error).
///
/// The full error can be extracted with [`casper_get_last_error`].  See [Error](super::Error) for
/// more details on what these mean.
```

Reply:
Whoops, I'm surprised I missed that. Thanks for catching that.